addons/l10n_tw_edi_ecpay/models/account_move_send.py

```diff
@@ -0,0 +1,117 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import json
+
+from odoo import SUPERUSER_ID, _, api, models
+
+
+class AccountMoveSend(models.AbstractModel):
+    _inherit = 'account.move.send'
+
+    @api.model
+    def _is_tw_edi_applicable(self, move):
+        return (move.is_invoice()
```

Comment:
is it expected to send vendor bills? if not
```suggestion
        return (move.is_sale_document()
```